pkgs/development/python-modules/fs/default.nix

```diff
@@ -30,13 +29,10 @@ buildPythonPackage rec {
 
   build-system = [ setuptools ];
 
-  buildInputs = [ glibcLocales ];
-
   dependencies = [
     six
     appdirs
     pytz
-    setuptools
```

Comment:
This change breaks some downstream packages; for example attempting to build `nomnatong` on current `master` (9e09ce2d47c7700b877becef99ce7e27f15604dc) results in

```
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/6kb205kz8m53bbgcs1xz5890zj4b6bad-source
source root is source/src
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
no configure script, doing nothing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
Traceback (most recent call last):
  File "/nix/store/csrwfkw01nys8knp67xhxhdg1n11jlwc-python3.12-afdko-4.0.1/bin/.makeotf-wrapped", line 6, in <module>
    from afdko.makeotf import main
  File "/nix/store/csrwfkw01nys8knp67xhxhdg1n11jlwc-python3.12-afdko-4.0.1/lib/python3.12/site-packages/afdko/makeotf.py", line 13, in <module>
    from afdko import convertfonttocid, fdkutils, ufotools
  File "/nix/store/csrwfkw01nys8knp67xhxhdg1n11jlwc-python3.12-afdko-4.0.1/lib/python3.12/site-packages/afdko/ufotools.py", line 13, in <module>
    from fontTools.ufoLib import UFOReader
  File "/nix/store/1pjjzg67xrnk5m6b053k23c0n0m3s03k-python3.12-fonttools-4.54.1/lib/python3.12/site-packages/fontTools/ufoLib/__init__.py", line 42, in <module>
    import fs
  File "/nix/store/xbk7nbi14ihmnjxdzphjxpifs3104j3d-python3.12-fs-2.4.16/lib/python3.12/site-packages/fs/__init__.py", line 4, in <module>
    __import__("pkg_resources").declare_namespace(__name__)  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named 'pkg_resources'
```

Reply:
That's a bug with those packages. `build-system = [ setuptools ];` should be added to those packages.

Reply:
The stacktrace literally points to this package...

Reply:
Fair enough. I'll submit a patch.